src/wavm/WAVMWasmModule.cpp

```diff
@@ -88,6 +88,9 @@ WAVMWasmModule::WAVMWasmModule()
 {
     stdoutMemFd = 0;
     stdoutSize = 0;
+
+    // Prepare OpenMP context cache
+    openMPContexts = std::vector<Runtime::Context*>(threadPoolSize, nullptr);
```

Comment:
I am not familiar with the `Runtime::Context` class, but do you think we could beneffit from using a smart pointer here?

Reply:
Unfortunately not as it's not created by us, WAVM returns us a raw pointer.